spec/controllers/trips_controller_spec.rb

```diff
@@ -24,4 +24,34 @@
       end
     end
   end
+
+  describe "DELETE /trips/:id" do
+    context "When trip exsist" do
+      it "destroys the trip" do
+        trip = create(:trip)
+
+        expect do
+          delete :destroy, params: { id: trip.id }
+        end.to change(Trip, :count).by(-1)
+      end
+
+      it "sets a flash notice message and redirects to index" do
+        trip = create(:trip)
+        notice = I18n.t("flash.actions.destroy.notice", resource_name: "Trip")
+
+        delete :destroy, params: { id: trip.id }
+
+        expect(flash[:notice]).to match(notice)
+        expect(response).to redirect_to trips_path
+      end
+    end
+
```

Comment:
Should the happy paths be in the feature specs?

Reply:
Also here I followed Elle suggestion for the delete#families and I used the same specs structure :)

Reply:
This spec should be separated into two specs with one assertion.
And yes, would be good to have a happy path/feature spec for deleting a trip from the UI

Reply:
- split the test with fixup in sha `733f331`;
- add feature specs for happy path in sha: `31d9e74`.